Data-Collection-Inventory/2021Q1_csvs/grant.csv

```diff
@@ -1,32 +1,20 @@
-U01 Chung grant,Towards Integrated 3D Reconstruction of Whole Human Brains at Subcellular Resolution,1U01MH117072-01,National Institute of Mental Health,Massachusetts Institute of Technology,9584926
 U01 Dong grant,Anatomical Characterization of Neuronal Cell Types of the Mouse Brain,1U01MH114829-01,National Institute of Mental Health,University of Southern California,9416006
 U01 Ecker grant,Epigenomic cell-type classification and regulatory element identification in the human brain,1U01MH121282-01,National Institute of Mental Health,Salk Institute for Biological Studies,9852895
 U01 Feng grant,A Molecular and Cellular Atlas of the Marmoset Brain,1U01MH114819-01,National Institute of Mental Health,Massachusetts Institute of Technology,9415538
 U01 Fischl grant,Imaging and analysis Techniques to Construct a Cell Census Atlas of the Human Brain,1U01MH117023-01,National Institute of Mental Health,Massachusetts General Hospital,9584061
 U01 Kriegstein grant,A Cellular Resolution Census of the Developing Human Brain,1U01MH114825-01,National Institute of Mental Health,University of California at San Francisco,9415877
 U01 Lein grant,A Multimodal Atlas of Human Brain Cell Types,1U01MH114812-01,National Institute of Mental Health,Allen Institute for Brain Science,9414852
 U01 Osten grant,Collaboratory for Atlasing Cell Type Anatomy in the Female and Male Mouse Brain,1U01MH114824-01,National Institute of Mental Health,Cold Spring Harbor Laboratory,9415873
-U01 Snyder-Mackler grant,Single cell transcriptional and epigenomic atlas of the macaque brain across the lifespan,1U01MH121260-01,National Institute of Mental Health,University of Washington,9852073
 U01 Yang grant,Dendritome Mapping of Genetically-Defined and Sparsely-Labeled Cortical and Striatal Projection Neurons,1U01MH117079-01,National Institute of Mental Health,University of California at Los Angeles,9585042
 U01 ZhangKun grant,Toward a Human Adult Brain Cell Atlas with Single-Cell Technologies,1U01MH114828-01A1,National Institute of Mental Health,University of California at San Diego,9584575
 U01 ZhangLi grant,Cell Atlas of Mouse Brain-Spinal Cord Connectome,1U01MH116990-01,National Institute of Mental Health,University of Southern California,9583948
 U19 Ecker grant,Center for Epigenomics of the Mouse Brain Atlas (CEMBA),1U19MH114831-01,National Institute of Mental Health,Salk Institute for Biological Studies,9416014
 U19 Huang grant,A Comprehensive Center for Mouse Brain Cell Atlas,1U19MH114821-01,National Institute of Mental Health,Cold Spring Harbor Laboratory,9415765
 U19 Zeng grant,A Comprehensive Whole-brain Atlas of Cell Types in the Mouse,1U19MH114830-01,National Institute of Mental Health,Allen Institute for Brain Science,9416007
-U24 Hawrylycz grant,A Community Resource for Single Cell Data in the Brain,1U24MH114827-01,National Institute of Mental Health,Allen Institute for Brain Science,9415946
```

Comment:
keep

Reply:
Added BICCN-related grants and subprograms that don't have associated projects yet.